pages/docs/reference/basic-syntax.md

```diff
@@ -602,3 +602,45 @@ fun main(args: Array<String>) {
 </div>
 
 See [Higher-order functions and Lambdas](lambdas.html).
+
+## Creating basic classes and its instances:
+
+<div class="sample" markdown="1">
+
+``` kotlin
+//sampleStart
+fun main(args: Array<String>) {
+    val rectangle: Rectangle = Rectangle (5.0, 2.0) //no 'new' keyword required in Kotlin while initializing
```

Comment:
It would be nice of you to re-format the code block, so that it follows the default code style that is used for Kotlin in IntelliJ IDEA and suggested by the [Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html). The readability would slightly benefit from it, and no one will stumble upon surprising code style.
You can do it automatically with IntelliJ IDEA.

Here are the points that might surprise the reader:
* The type and function names should not be separated from the opening parenthesis by a space (e.g. `Rectangle (5.0, 2.0)` → `Rectangle(5.0, 2.0)`), and the arguments are not (e.g. `print( "..." )` → `print("...")`).
* Also, please surround all binary operators including `*` and `/` with whitespaces (e.g. `s*(s - sideA)*(s - sideB)*(s - sideC)` → `s * (s - sideA) * (s - sideB) * (s - sideC)`).
* The Kotlin naming conventions also suggest using *UpperCamelCase* for type names (`interface rectangleProperties` → `interface RectangleProperties`).
* And the last, the colon `:` should be prepended by whitespace if and only if it means subtyping relationship (`class Triangle(...): Shape(3)` → `class Triangle(...) : Shape(3)`.